README.md

```diff
@@ -2375,39 +2372,34 @@ source ~/.bash_completion.d/_zapier
 
 Finally, restart your shell and start hitting TAB with the `zapier` command!
 
-## Upgrading Zapier Platform CLI or Zapier Platform Core
+## The Zapier Platform Packages
```

Comment:
I'm having a little trouble following what you're recommending here. Is the problem with the highlighted line or the content of the section in general? 

I think part of ~our~ my (don't want to speak for everyone) problem here is that we've worked extensively with these packages (and node in general) so a lot of the necessary actions here seem very obvious. Since CLI and core are updated separately (directly from npm vs editing a file and re-installing) it made more sense to me to thematically break them out a bit. Also thinking about the user with multiple apps, CLI will be updated once but core will be updated repeatedly, so being able to access those separately in the docs made sense.

Reply:
Sorry that wasn't clear! My comment was related to the content of the section in general.

I agree with you on the experience with node, but we've seen developers get confused about `core` vs `cli`, why each existed, and how/where to update each.

If we've found out this isn't the case anymore, or that this new section caused more confusion, I'm OK with it being changed/removed, otherwise I'd prefer to keep it.

Reply:
Nothing to apologize for. 😄The goal here was to clean up the language, the content itself was good.